.travis.yml

```diff
@@ -7,24 +7,16 @@ cache:
   directories:
     - node_modules
 
-branches:  # pushes only get built for master
-  only:
-    - master
```

Comment:
>Overall this PR LGTM, but I'd keep the only: master build - I can't think of a reason why we would be interested in push-only builds for this repo.

I'd like to keep all our repos the same otherwise differences can be potential side effects for other tools, like Coveralls.